open-metadata-implementation/user-interfaces/presentation-server/cra-server/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "cra-server",
```

Comment:
is this the final name?

Reply:
updated names to `presentation-server` and `presentation-server-client`